tests/e2e/test_formatting.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import os
+
+
+def are_contents_equal(filepath1: str, filepath2: str):
+    with open(filepath1, "r", encoding="utf-8") as f1:
+        with open(filepath2, "r", encoding="utf-8") as f2:
+            return f1.read() == f2.read()
```

Comment:
```suggestion
def assert_files_equal(filepath1: str, filepath2: str):
    assert Path(filepath1).read_text() == Path(filepath2).read_text()
```

1. shorter :D 
2. because equality is done directly in `assert` statement, pytest will be able to output diff in logs

Reply:
I couldn't find a way to extract this nicely with assert because we need to negate the condition in one of the tests.

Reply:
@Radinyn please use `assert_*` approach, due to the benefit outlined in point 2